inductiva/tasks/task.py

```diff
@@ -155,3 +156,22 @@ class _PathParams(TypedDict):
 
     def _get_path_params(self) -> _PathParams:
         return {"task_id": self.id}
+
+    def get_time(self) -> float:
```

Comment:
maybe you want to actually name this get_execution_time?
Btw: should you also have other methods related to time? e.g. get_X_time where X can be (@luispcunha?):

* start
* end
* dispatch
*

Reply:
I see some potential for this, but there is no foreseeable use for them now.. It will just basically be replicating what David did below.

What we could obtain would be machine information that is also saved inside the task :)

Reply:
Those times are already available to the user, the only difference this method brings is that it computes the difference between two of them (end and start).